### ISSUE TITLE
`get_variable_def` does not work for  stem density,`"en"`

### ISSUE THREAD

**Issue by brycefrank** [allometric/allometric#189]
```
[ins] r$> allometric::get_variable_def("en")
Error in data.frame(..., check.names = FALSE) : 
  arguments imply differing number of rows: 0, 1
In addition: Warning message:
In allometric::get_variable_def("en") :
  No variable definition found for: en
```

**Comment by brycefrank** [comment]
Turns out `"en"` is depricated, and `"es"` should be used instead. This tiggers a refactor in `models` first, then resolve this issue.

**Comment by brycefrank** [comment]
Resolved in [e88be95](https://github.com/allometric/allometric/commit/e88be958386b8b27c93c7bd90e885c819c1c8b9b)
